Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -56,7 +56,10 @@ public struct TuistCommand: ParsableCommand {
             _exit(exitCode(for: error).rawValue)
         }
         do {
-            try command.run()
+            var trackableCommand = TrackableCommand(command: command)
+            try trackableCommand.run(completion: { _ in
```

Comment:
This architecture allows me to not change any specific command

Reply:
Yep that's nice 👍 Elegant solution 👏